product-service/src/handlers/getProductsById.ts

```diff
@@ -0,0 +1,31 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+
+import { productsList } from '../mockProductsList';
+
+export const getProductsById: APIGatewayProxyHandler = async event => {
+  try {
+    const { productId } = event.pathParameters;
+
+    const productById: Product = productsList.find(product => product.id === productId);

```

Comment:
As I understand `Product` type is redundant. Because `productList` is array of `Product[]` and `find` will return `Product` in that case.

Reply:
@AndreyS55 Thanks, you are right, I've made changes